docker-compose.yml

```diff
@@ -7,14 +7,18 @@ services:
     working_dir: ${PROJECT_PATH}
     command: bash -c "glide init --non-interactive"
   dev:
+    build: .
     image: flowup/godev
     volumes:
      - .:${PROJECT_PATH}:Z
     working_dir: ${PROJECT_PATH}
-    command: bash -c "glide install && go install ${CMD_GOPATH} && ${PROJECT_CMD_NAME} 'echo \"some file was changed\"'"
+    command: bash -c "glide install && go install ${CMD_GOPATH} && ${PROJECT_CMD_NAME} -r 'for i in {1..10}; do echo \"hello $i\" ; sleep 1 ;done; echo \"----------\" ' --ignore \"vendor\" --ignore \".git\" --ignore \".glide\" -v"
```

Comment:
did you suggest delete
    `command: bash -c "glide install && go install ${CMD_GOPATH} && ${PROJECT_CMD_NAME} -r 'for i in {1..10}; do echo \"hello $i\" ; sleep 1 ;done; echo \"----------\" ' --ignore \"vendor\" --ignore \".git\" --ignore \".glide\" -v"`

and let there this ?
`command: bash -c "glide install && goconvey -timeout 5s -host 0.0.0.0"`